cobra/flux_analysis/loopless.py

```diff
@@ -75,11 +75,11 @@ def add_loopless(model, zero_cutoff=1e-12):
         name = "nullspace_constraint_" + str(i)
         nullspace_constraint = prob.Constraint(S.Zero, lb=0, ub=0, name=name)
         add_to_solver(model, [nullspace_constraint])
```

Comment:
I think we should standardize how we want to call these functions, `add_to_solver(model, ...)` vs. `model.add_to_problem(...)`